packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx

```diff
@@ -45,6 +46,11 @@ interface Props {
     onRemove: () => void;
     value: FormState;
     withNewTabOption: boolean;
+    infoText?: {
+        text: string;
+        actionLabel?: string;
+        action?: () => void;
+    };
```

Comment:
I have a bad feeling about this interface. I think it's too much tailored to the use-case to the point it's almost hard-coded.

How about changing it to something more generic:

```ts
info: Array<string | { text: string, href: string } | { text: string, onClick: () => void }>
```

And then pass `info` as:

```ts
info={[
    "Your current theme doesn't support these options. ", 
    { text: 'Go to themes', onClick: ... },
]} 
```

what do you think?

Reply:
I agree with this, making the update

Reply:
Updated